geode-core/src/main/java/org/apache/geode/distributed/internal/membership/gms/GMSMemberData.java

```diff
@@ -95,7 +97,7 @@ public GMSMemberData(String localhost, int udpPort, Version version) {
    * @param p the membership listening port
    */
   @VisibleForTesting
-  public GMSMemberData(String i, int p) {
+  public GMSMemberData(String i, int p) throws UnknownHostException {
```

Comment:
I don't see this constructor being used in my intellij project...

Reply:
removed.